client/signup/jetpack-connect/plans.jsx

```diff
@@ -12,7 +12,7 @@ import { bindActionCreators } from 'redux';
 import { getPlansBySite } from 'state/sites/plans/selectors';
 import { getFlowType } from 'state/jetpack-connect/selectors';
 import Main from 'components/main';
-import ConnectHeader from './connect-header';
+import StepHeader from '../step-header';
```

Comment:
I think it's not especiffically mentioned in our style guide. Folks used to do it when we used `require` instead of `import`, but as @tyxla say, now it's a little more complex. 

Maybe we should have at least some broad stroke agreement about how to order them (ping @mtias @aduth  )